### ISSUE TITLE
2023.5.0b0 New Error

### ISSUE THREAD

**Issue by DavidFW1960** [exxamalte/home-assistant-custom-components-nsw-rural-fire-service-fire-danger#61]
```
Error setting up entry Greater Sydney Region for nsw_rural_fire_service_fire_danger
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/nsw_rural_fire_service_fire_danger/__init__.py", line 90, in async_setup_entry
    data_feed_coordinator = NswRfsFireDangerExtendedFeedCoordinator(hass, entry)
  File "/config/custom_components/nsw_rural_fire_service_fire_danger/coordinator.py", line 52, in __init__
    self._rest = RestData(
TypeError: RestData.__init__() missing 1 required positional argument: 'ssl_cipher_list'
```

**Comment by exxamalte** [comment]
Oh, no, not again 😵‍💫
Thanks for reporting this, I'll see if I can work around this again, or have to finally implement my own data fetching service.

**Comment by DavidFW1960** [comment]
It's almost like they deliberately want to screw people over

**Comment by exxamalte** [comment]
This is now fixed in v0.16
